iceoryx_posh/source/roudi/port_manager.cpp

```diff
@@ -26,10 +26,10 @@ namespace iox
 {
 namespace roudi
 {
-capro::Interfaces StringToEInterfaces(std::string f_str)
+capro::Interfaces StringToEInterfaces(std::string str)
```

Comment:
❤️

Reply:
I'm watching https://www.youtube.com/watch?v=MBRoCdtZOYg right now, so why not give it a more descriptive name like `caproInterface`
... and why is this called `StringToEInterfaces` when there is no `EInterface` anymore?

Reply:
I just wanted to get rid of f_ . You're dialing another number ;-)

Reply:
You touched it, now you have to change the argument type to: `const std::string & str` ;) ... Please

Reply:
OK, will touch it again ;-)